examples/POS-client/ERC1155/burnSingle.js

```diff
@@ -4,11 +4,13 @@ const maticPOSClient = utils.getMaticPOSClient()
 
 const execute = async () => {
   try {
-    const tx = await maticPOSClient.burnSingleERC1155(config.child.DERC1155, config.user.tokenId, config.user.amount)
-    console.log(tx.transactionHash) // eslint-disable-line
+    const tx = await maticPOSClient.isSingleERC1155ExitProcessed(
```

Comment:
previous example was for `burnSingleERC1155` ? why we made to `isSingleERC1155ExitProcessed `

Reply:
don't know how the files got rearranged. fixed